README.md

```diff
@@ -116,10 +116,19 @@ The plugin provides two commands, `addNewSnippet` and `editSnippet`. Here is the
 code to create keymaps for them:
 
 ```lua
-vim.keymap.set("n", "<leader>se", function() require("scissors").editSnippet() end)
+vim.keymap.set("n", "<leader>se", require("scissors").editSnippet)
 
 -- When used in visual mode prefills the selection as body.
-vim.keymap.set({ "n", "x" }, "<leader>sa", function() require("scissors").addNewSnippet() end)
+vim.keymap.set({ "n", "x" }, "<leader>sa", require("scissors").addNewSnippet)
+```
+
+And here is the code to create user commands:
+
+```lua
```

Comment:
Having the user create user commands on their own feels somewhat unnecessary. I think it would be more user-friendly to have the plugin create the commands for the user.

I'd suggest creating a file `./plugin/user-commands.lua` and simply adding those two user commands there. Also, when doing so:
- wrap the `require` calls for lazy-loading ;)
- Use a name like `ScissorsAddSnippet` and `ScissorsEditSnippet`, since I think it's common practice to make it transparent to the user which plugin added the command